.travis.yml

```diff
@@ -38,13 +38,22 @@ matrix:
     - php: 7.4
       env:
         - DEPS=latest
+    - php: 7.4
+      env:
+        - DEPS=lowest
+        - COMPOSER_VERSION=2
```

Comment:
Should we also force composer version 1 in case 2 becomes the standard?

Reply:
Mh, good point!